spec/factories/clearance.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
```

Comment:
Is `factories/clearance.rb` the best home for this, you think?  Would `factories/user.rb` be better (if you're going the many factory files route)?

Definitions are [picked up either way](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#defining-factories)